rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -27,21 +27,12 @@ THE SOFTWARE.
 
 FusedCropTJDecoder::FusedCropTJDecoder(){
     m_jpegDecompressor = tjInitDecompress();
-
-#if 0
-    int num_avail_scalings = 0;
-    auto scaling_factors = tjGetScalingFactors	(&num_avail_scalings);	
-    for(int i = 0; i < num_avail_scalings; i++) {
-        if(scaling_factors[i].num < scaling_factors[i].denom) {
-
-            printf("%d / %d  - ",scaling_factors[i].num, scaling_factors[i].denom );
-        }
-    }
-#endif
+    //generate BatchSize of RNG's- Using a random seed
+    int64_t seed = getseed();
+    generate_rngs(seed, 256);
```

Comment:
I think generate_rngs() is not implemented correct. What is the purpose of this func. Why is hardcoded to 256